src/BigInteger/AdapterPluginManager.php

```diff
@@ -21,13 +22,23 @@
 class AdapterPluginManager extends AbstractPluginManager
 {
     /**
-     * Default set of adapters
+     * Default set of adapter aliases
      *
-     * @var array
+     * @var string[]
      */
-    protected $invokableClasses = [
-        'bcmath' => 'Zend\Math\BigInteger\Adapter\Bcmath',
-        'gmp'    => 'Zend\Math\BigInteger\Adapter\Gmp',
+    protected $aliases = [
+        'Bcmath' => 'Zend\Math\BigInteger\Adapter\Bcmath',
+        'Gmp'    => 'Zend\Math\BigInteger\Adapter\Gmp',
+    ];
```

Comment:
The reason for this is that it already uses the proper case: https://github.com/zendframework/zend-math/blob/master/src/BigInteger/BigInteger.php#L97

We could add an additional alias for the lowercase version which might not be a bad idea... thoughts?